appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">dolphin-emu.desktop</id>
+  <name>PrimeHack</name>
+  <summary>Dolphin Emu fork for various Metroid Prime mods</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <content_rating type="oars-1.0" />
+  <description><p>PrimeHack is a fork of Dolphin Emulator to bring modern Mouse & Keyboard controls, as well as Dual-Stick gamepad controls to the Metroid Prime Trilogy. PrimeHack also offers many other features such as increased FoV and various new cheats.</p></description>
+  <screenshots>
+    <screenshot type="default">https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/1.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/2.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/3.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/4.png</screenshot>
+  </screenshots>
```

Comment:
so it still display Dolphin screenshots?

Reply:
The UI isn't significantly different. The *controls*, are. I didn't feel it would be particularly necessary to show screenshots of the game, or our extra settings tabs. This is a fork, after-all.

Reply:
I think it should use separate screenshots nonetheless.